.github/validation_samples/it_pileup/config.py

```diff
@@ -61,9 +61,17 @@
         TrigScintDigiProducer.pad2(),
         TrigScintDigiProducer.pad3(),
         ]
-for d in ts_digis :
-    d.randomSeed = 1
-    d.input_collection += overlayStr
+for digi in ts_digis :
+    digi.input_collection += overlayStr
+
+ts_clusters = [
+        TrigScintClusterProducer.pad1(),
+        TrigScintClusterProducer.pad2(),
+        TrigScintClusterProducer.pad3(),
+        ] 
+for clu in ts_clusters :
+    clu.input_collection += overlayStr
```

Comment:
are you sure this is working? is `overlayStr` appended to the output collection of the digi step? otherwise it shouldn't be able to find it...

Reply:
I'm not sure but 1) this is how it's been in this config, 2) I'd expect that the CI would fail if this doesnt exists, right?

Reply:
>  this is how it's been in this config

on this I mean that the original config never appended the overlayStr to the digi output, maybe that's been wrong all along. Still confusing why 2) would work

Reply:
i think you are right about 2) but i am also confused.

Reply:
oh but the CI did fail.

```
  [ fire ] 1 fatal: [ProductNotFound] : No product found for name 'trigScintDigisPad1Overlay'
    at /home/runner/work/ldmx-sw/ldmx-sw/Framework/include/Framework/Event.h:292 in getObject
```

Reply:
there are two ways to get the right collection.
- set a new output name from digi
- leave as is but use the pass name of the overlay process for the clustering input

Looking at the cluster plots you attached to the PR, one could suspect that they didn't contain pileup before? Considering the TS granularity, in 1/24~4% of the 2e cases you should see 2 MIPs per cluster which should cause extra bumps in the cluster PE distribution.

Reply:
Ok... that whole config is wrong, almost everything is taking the 1e instead of the 2e. I'll fix it in a next PR, for now I just made this functional